keyboards/maxr1998/phoebe/phoebe.h

```diff
@@ -14,5 +14,5 @@
       { k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b }, \
       { k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b }, \
       { k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b }, \
-      { k40, k41, k42, k43, k44,    k45,   k47, k48, k49, k4a, k4b } \
+      { k40, k41, k42, k43, k44, k45, XXXXXXX, k47, k48, k49, k4a, k4b } \
```

Comment:
```suggestion
      { k40, k41, k42, k43, k44, k45, KC_NO, k47, k48, k49, k4a, k4b } \
```

Reply:
Oops, I still wanted to fix that, thanks for the reminder. Shouldn't just repeating the space keycode also work?

Reply:
> Shouldn't just repeating the space keycode also work?

It does, but QMK custom here is to use `KC_NO` for matrix positions without a switch in the physical layout. If your 2u Space key is at `k45` and there isn't a switch at `k46`, replacing `k46` with `KC_NO` is the preferred way to note that.